src/mem/mem_ctrl.hh

```diff
@@ -139,7 +142,7 @@ class MemPacket
      * A pointer to the BurstHelper if this MemPacket is a split packet
      * If not a split packet (common case), this is set to NULL
      */
-    BurstHelper* burstHelper;
+    BurstHelper *burstHelper;
```

Comment:
Ok, this is actually the last time I'm going to comment on a non-functional change :)